pep-0458.txt

```diff
@@ -79,23 +85,25 @@ Before the wiki attack, PyPI used MD5 hashes to tell package managers, such as
 pip, whether or not a package was corrupted in transit.  However, the absence
 of SSL made it hard for package managers to verify transport integrity to PyPI.
 It was therefore easy to launch a man-in-the-middle attack between pip and
-PyPI, and change package content arbitrarily.  Users could be tricked into
-installing malicious packages with man-in-the-middle attacks.  After the wiki
+PyPI, and change package content arbitrarily.  As a result, users could be tricked into
+installing malicious packages.  After the wiki
 attack, several steps were proposed (some of which were implemented) to deliver
-a much higher level of security than was previously the case: requiring SSL to
+a much higher level of security than was previously the case. These steps included
+requiring SSL to
 communicate with PyPI [6]_, restricting project names [7]_, and migrating from
 MD5 to SHA-2 hashes [8]_.
 
-These steps, though necessary, are insufficient because attacks are still
+Though necessary, these steps are insufficient to protect package because attacks are still
```

Comment:
_these steps are insufficient to protect [a] package_ / _to protect [packages]_.